contracts/governance/Staking/Staking.sol

```diff
@@ -90,32 +93,78 @@ contract Staking is WeightedStaking{
      * @param amount the amount of SOV tokens
      * @param stakeFor the address for which we want to increase the stake. staking for the sender if 0x0
      * @param until the lock date until which the funds are staked
-     */
-    function increaseStake(uint96 amount, address stakeFor, uint until) public{
+     * */
+    //TODO deprecated, can be removed when frontend updated
```

Comment:
I think we can remove it immediatley. i'll let victor know that the next staking contract will no longer support this function.

Reply:
done